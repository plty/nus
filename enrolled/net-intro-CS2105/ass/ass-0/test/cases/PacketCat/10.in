Size: 1B8Size: 0BSize: 1BcSize: 1BXSize: 1B6Size: 0BSize: 1BFSize: 0BSize: 1BhSize: 1BHSize: 0BSize: 1BOSize: 1BlSize: 0BSize: 1B5Size: 1BGSize: 1BaSize: 0BSize: 1BDSize: 0BSize: 0BSize: 1BuSize: 1BCSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B1Size: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B6Size: 1BBSize: 0BSize: 1BbSize: 1B7Size: 0BSize: 0BSize: 1BlSize: 0BSize: 1BCSize: 1B2Size: 1BUSize: 0BSize: 0BSize: 1B3Size: 0BSize: 0BSize: 1BdSize: 1BJSize: 0BSize: 0BSize: 0BSize: 1B0Size: 0BSize: 1BzSize: 0BSize: 0BSize: 0BSize: 1BQSize: 1BkSize: 1BrSize: 1BASize: 1B9Size: 0BSize: 0BSize: 1BISize: 0BSize: 0BSize: 1BtSize: 1B4Size: 1B+Size: 1B3Size: 0BSize: 1BDSize: 1B0Size: 1BJSize: 0BSize: 1BPSize: 0BSize: 1B3Size: 0BSize: 0BSize: 0BSize: 1BRSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BnSize: 0BSize: 0BSize: 0BSize: 1BGSize: 1B5Size: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B+Size: 0BSize: 0BSize: 1BwSize: 0BSize: 0BSize: 1BtSize: 0BSize: 1BNSize: 1BlSize: 1BaSize: 0BSize: 0BSize: 1B7Size: 0BSize: 0BSize: 0BSize: 0BSize: 1BeSize: 1BtSize: 1BySize: 0BSize: 1BkSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BKSize: 0BSize: 1BUSize: 0BSize: 1BgSize: 1BmSize: 1BXSize: 0BSize: 0BSize: 0BSize: 1BuSize: 1BmSize: 0BSize: 0BSize: 1BhSize: 0BSize: 0BSize: 0BSize: 1BvSize: 0BSize: 0BSize: 1B7Size: 0BSize: 1BpSize: 1B7Size: 1BuSize: 1BTSize: 0BSize: 0BSize: 1B0Size: 0BSize: 1B/Size: 0BSize: 1BuSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BTSize: 0BSize: 1BJSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BgSize: 1BhSize: 0BSize: 0BSize: 1BmSize: 1BBSize: 0BSize: 0BSize: 0BSize: 1BzSize: 1B6Size: 1BXSize: 1BXSize: 0BSize: 1BOSize: 0BSize: 1BySize: 1BUSize: 1BzSize: 0BSize: 0BSize: 0BSize: 1BzSize: 0BSize: 1BuSize: 1BfSize: 1BdSize: 0BSize: 0BSize: 1B5Size: 1BESize: 1BVSize: 1BlSize: 1B4Size: 0BSize: 1BsSize: 1BdSize: 0BSize: 0BSize: 1BSSize: 1BjSize: 0BSize: 0BSize: 0BSize: 1BASize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BeSize: 1BMSize: 1B/Size: 0BSize: 0BSize: 1BpSize: 1BLSize: 1BsSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BgSize: 1B+Size: 0BSize: 0BSize: 1BWSize: 1B2Size: 0BSize: 1BySize: 1BHSize: 1BySize: 1B7Size: 0BSize: 0BSize: 1B/Size: 1BJSize: 1BNSize: 1BiSize: 0BSize: 1BASize: 0BSize: 1BSSize: 1BsSize: 0BSize: 1BISize: 1BSSize: 1BPSize: 1B4Size: 0BSize: 1BUSize: 1BySize: 0BSize: 1BPSize: 0BSize: 0BSize: 0BSize: 1BrSize: 0BSize: 0BSize: 1BoSize: 0BSize: 1BrSize: 0BSize: 1BQSize: 0BSize: 0BSize: 1BdSize: 1BpSize: 1BYSize: 1B9Size: 0BSize: 0BSize: 0BSize: 0BSize: 1BeSize: 1BrSize: 1BNSize: 1BVSize: 0BSize: 1BASize: 1BPSize: 1BISize: 1BGSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B1Size: 1BDSize: 0BSize: 0BSize: 1BISize: 0BSize: 1BNSize: 1BSSize: 0BSize: 0BSize: 1BySize: 0BSize: 1BQSize: 1B1Size: 0BSize: 0BSize: 0BSize: 1BuSize: 1BoSize: 0BSize: 1BASize: 1BzSize: 0BSize: 1B7Size: 0BSize: 0BSize: 0BSize: 0BSize: 1BCSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BRSize: 0BSize: 0BSize: 1BhSize: 1B5Size: 0BSize: 1B0Size: 1B8Size: 1BRSize: 0BSize: 0BSize: 1BDSize: 0BSize: 1BeSize: 1BvSize: 1BFSize: 1BGSize: 1B/Size: 0BSize: 1BmSize: 0BSize: 1BpSize: 1BUSize: 1BySize: 1BcSize: 1BRSize: 0BSize: 0BSize: 1B5Size: 0BSize: 1BeSize: 1B1Size: 0BSize: 0BSize: 1BxSize: 1BZSize: 1B2Size: 1BfSize: 1BRSize: 0BSize: 0BSize: 1BOSize: 0BSize: 1BGSize: 1B2Size: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BASize: 0BSize: 1B7Size: 0BSize: 0BSize: 1BsSize: 0BSize: 1BLSize: 1BCSize: 1B1Size: 0BSize: 0BSize: 1BrSize: 1BySize: 1BaSize: 1BmSize: 1B1Size: 1B9Size: 1BsSize: 0BSize: 1BvSize: 1BUSize: 1B5Size: 0BSize: 0BSize: 0BSize: 0BSize: 1BxSize: 0BSize: 0BSize: 1BcSize: 0BSize: 1BWSize: 0BSize: 1BSSize: 0BSize: 0BSize: 0BSize: 1B4Size: 0BSize: 1B5Size: 0BSize: 1BTSize: 0BSize: 1BzSize: 1BSSize: 0BSize: 1BHSize: 0BSize: 1BOSize: 0BSize: 0BSize: 1BQSize: 1BWSize: 0BSize: 1BNSize: 1BbSize: 1BwSize: 0BSize: 0BSize: 0BSize: 1BVSize: 0BSize: 1BmSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B7Size: 1BVSize: 1BJSize: 1BZSize: 1BhSize: 0BSize: 1BQSize: 1B0Size: 1BmSize: 0BSize: 1BESize: 0BSize: 1BUSize: 1BaSize: 1BlSize: 0BSize: 1BtSize: 1BDSize: 1BISize: 0BSize: 0BSize: 0BSize: 1BPSize: 1BbSize: 0BSize: 0BSize: 0BSize: 1B0Size: 0BSize: 1BmSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B9Size: 0BSize: 1BISize: 0BSize: 0BSize: 1BNSize: 1B8Size: 0BSize: 0BSize: 0BSize: 1BeSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BYSize: 0BSize: 1BBSize: 1B3Size: 1BjSize: 1BbSize: 1BXSize: 0BSize: 1BISize: 1BcSize: 1BESize: 1BKSize: 0BSize: 1BSSize: 1B2Size: 1BiSize: 0BSize: 1BZSize: 0BSize: 0BSize: 0BSize: 1BRSize: 1BlSize: 1B/Size: 0BSize: 1BUSize: 1BtSize: 0BSize: 1BESize: 1BvSize: 1B4Size: 1BvSize: 0BSize: 1B1Size: 1BOSize: 0BSize: 1BeSize: 0BSize: 1BSSize: 1BlSize: 1BjSize: 0BSize: 0BSize: 1BdSize: 1BySize: 1BuSize: 0BSize: 0BSize: 0BSize: 1BESize: 0BSize: 1B4Size: 0BSize: 0BSize: 1BsSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BbSize: 1BgSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BHSize: 1BzSize: 1BvSize: 0BSize: 1BXSize: 1BYSize: 1BwSize: 0BSize: 0BSize: 1BISize: 0BSize: 0BSize: 0BSize: 0BSize: 1BgSize: 1B7Size: 1BwSize: 1BKSize: 0BSize: 1BKSize: 1B9Size: 0BSize: 0BSize: 1BWSize: 1BuSize: 0BSize: 0BSize: 1BsSize: 1BHSize: 0BSize: 0BSize: 0BSize: 1BfSize: 0BSize: 1BMSize: 1BYSize: 1BASize: 0BSize: 1BASize: 0BSize: 0BSize: 0BSize: 0BSize: 1BoSize: 1BPSize: 0BSize: 1BESize: 0BSize: 0BSize: 0BSize: 1BqSize: 1BdSize: 1B/Size: 1BuSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B8Size: 0BSize: 1BASize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BcSize: 0BSize: 1BnSize: 1BASize: 1BMSize: 1BsSize: 0BSize: 1BzSize: 1B2Size: 0BSize: 0BSize: 0BSize: 1BJSize: 0BSize: 0BSize: 0BSize: 1BYSize: 1BBSize: 1BSSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B4Size: 0BSize: 1BMSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B6Size: 1BoSize: 1BuSize: 1BMSize: 1BASize: 0BSize: 1BlSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BbSize: 1BKSize: 0BSize: 1BuSize: 0BSize: 0BSize: 0BSize: 1BwSize: 1B8Size: 0BSize: 1B/Size: 1BDSize: 1BRSize: 0BSize: 1BKSize: 0BSize: 1BISize: 1B9Size: 1BFSize: 0BSize: 0BSize: 1BBSize: 0BSize: 0BSize: 1BeSize: 1BWSize: 0BSize: 1BCSize: 1BCSize: 0BSize: 1BBSize: 1BmSize: 0BSize: 0BSize: 1BWSize: 0BSize: 1BiSize: 1B0Size: 1BYSize: 0BSize: 0BSize: 1BhSize: 0BSize: 1BmSize: 1B8Size: 1B/Size: 0BSize: 0BSize: 0BSize: 1BGSize: 0BSize: 0BSize: 1BrSize: 0BSize: 1BfSize: 0BSize: 1B1Size: 0BSize: 0BSize: 1BxSize: 0BSize: 1BmSize: 1BHSize: 0BSize: 1B2Size: 0BSize: 1BCSize: 0BSize: 1BYSize: 0BSize: 1BBSize: 1BtSize: 0BSize: 0BSize: 1B9Size: 0BSize: 0BSize: 0BSize: 1BwSize: 1BVSize: 1BNSize: 1BrSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BCSize: 1BASize: 0BSize: 0BSize: 1BdSize: 1BlSize: 1BaSize: 1BYSize: 1BOSize: 1BfSize: 1BxSize: 0BSize: 1BVSize: 0BSize: 0BSize: 1BiSize: 1BjSize: 0BSize: 1BpSize: 1BiSize: 1BVSize: 0BSize: 1BJSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B7Size: 1B5Size: 1BKSize: 1BxSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BkSize: 0BSize: 1BzSize: 0BSize: 0BSize: 1BPSize: 0BSize: 0BSize: 0BSize: 1BbSize: 1BESize: 0BSize: 1BtSize: 0BSize: 0BSize: 1B+Size: 0BSize: 0BSize: 1BOSize: 1B6Size: 0BSize: 1B0Size: 1BrSize: 0BSize: 1BiSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B0Size: 0BSize: 1BvSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BbSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BjSize: 1BJSize: 1BZSize: 0BSize: 1B3Size: 0BSize: 0BSize: 0BSize: 1BrSize: 1BYSize: 1BtSize: 0BSize: 1BhSize: 0BSize: 1B7Size: 1B9Size: 0BSize: 1BXSize: 0BSize: 1BiSize: 0BSize: 1BOSize: 0BSize: 1BFSize: 1B7Size: 1BySize: 1BiSize: 0BSize: 0BSize: 0BSize: 1BdSize: 1BeSize: 1BlSize: 1BFSize: 0BSize: 1BrSize: 1BqSize: 1BcSize: 1B0Size: 1BFSize: 0BSize: 0BSize: 1BxSize: 0BSize: 0BSize: 0BSize: 1BeSize: 1BzSize: 1BbSize: 0BSize: 0BSize: 0BSize: 0BSize: 1BaSize: 0BSize: 1B8Size: 0BSize: 1BeSize: 0BSize: 1BtSize: 0BSize: 1BhSize: 1BySize: 0BSize: 0BSize: 1BTSize: 1BOSize: 0BSize: 1BLSize: 0BSize: 1BcSize: 1BjSize: 0BSize: 0BSize: 1BOSize: 0BSize: 1B7Size: 1B5Size: 1BnSize: 0BSize: 1BXSize: 1BLSize: 1BzSize: 1BwSize: 1BrSize: 1BdSize: 0BSize: 1BcSize: 0BSize: 1BHSize: 0BSize: 0BSize: 1B7Size: 0BSize: 1BhSize: 1B6Size: 0BSize: 1BqSize: 0BSize: 1BWSize: 1BBSize: 0BSize: 0BSize: 0BSize: 0BSize: 0BSize: 1B5Size: 0BSize: 1BUSize: 1BfSize: 1BSSize: 1BLSize: 0BSize: 1BgSize: 0BSize: 1BSSize: 1BRSize: 1BASize: 0BSize: 1BDSize: 0BSize: 0BSize: 1BxSize: 1B5Size: 0BSize: 1BDSize: 0BSize: 0BSize: 1BNSize: 1BlSize: 1BASize: 0BSize: 0BSize: 1BaSize: 0BSize: 0BSize: 1B3Size: 0BSize: 0BSize: 1B4Size: 0BSize: 0BSize: 1B7Size: 1BN