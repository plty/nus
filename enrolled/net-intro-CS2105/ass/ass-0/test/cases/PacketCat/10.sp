840 48
1684 92
2540 148
3390 198
4240 248
5093 301
5936 344
6788 396
7634 442
8486 494
