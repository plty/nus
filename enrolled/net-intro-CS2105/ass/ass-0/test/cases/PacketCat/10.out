8cX6FhHOl5GaDuC16Bb7lC2U3dJ0zQkrA9It4+3D0JP3RnG5+wtNla7etykKUgmXumhv7p7uT0/uTJghmBz6XXOyUzzufd5EVl4sdSjAeM/pLsg+W2yHy7/JNiASsISP4UyProrQdpY9erNVAPIG1DINSyQ1uoAz7CRh508RDevFG/mpUycR5e1xZ2fROG2A7sLC1ryam19svU5xcWS45TzSHOQWNbwVm7VJZhQ0mEUaltDIPb0m9IN8eYB3jbXIcEKS2iZRl/UtEv4v1OeSljdyuE4sbgHzvXYwIg7wKK9WusHfMYAAoPEqd/u8AcnAMsz2JYBS4M6ouMAlbKuw8/DRKI9FBeWCCBmWi0Yhm8/Grf1xmH2CYBt9wVNrCAdlaYOfxVijpiVJ75KxkzPbEt+O60ri0vbjJZ3rYth79XiOF7yidelFrqc0Fxezba8ethyTOLcjO75nXLzwrdcH7h6qWB5UfSLgSRADx5DNlAa347N